panel/src/components/Views/LoginView.vue

```diff
@@ -3,21 +3,19 @@
     {{ issue.message }}
   </k-error-view>
   <k-view v-else-if="ready && form === 'login'" align="center" class="k-login-view">
-    <k-login-form />
+    <k-plugin-login />
```

Comment:
Could we name this `<k-login-plugin>` instead?

Reply:
@bastianallgeier Done. ✅